engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -66,25 +71,10 @@ public ModuleManager(String masterServerAddress) {
     }
 
     public ModuleManager(String masterServerAddress, List<Class<?>> classesOnClasspathsToAddToEngine) {
-        PathManager pathManager = PathManager.getInstance();  // get early so if it needs to initialize, it does it now
```

Comment:
I'm sure there was _something_ that prompted me to do this with the comment, but now I can't imagine what that would have been. Instantiating PathManager doesn't have any effects that can be seen without doing this same `getInstance` call. e.g. it does _not_ change the process's working directory.

So we'll pretend I'm better-informed now than I was then and remove it.